command/sidechain/validators/validator_info.go

```diff
@@ -47,7 +56,13 @@ func runCommand(cmd *cobra.Command, _ []string) error {
 	outputter := command.InitializeOutputter(cmd)
 	defer outputter.WriteOutput()
 
-	validatorAccount, err := sidechainHelper.GetAccountFromDir(params.accountDir)
+	// get secret manager and allow reading from local directory (true at the end)
```

Comment:
Maybe this should be inside a helper function. The same boilerplate code is being used more than a few times.:
```
// get secret manager and allow reading from local directory (true at the end)
	secretsManager, err := polybftsecrets.GetSecretsManager(params.accountDir, params.configPath, true)
	if err != nil {
		return err
	}

	validatorAccount, err := wallet.NewAccountFromSecret(secretsManager)
	```

Reply:
fixed: https://github.com/0xPolygon/polygon-edge/pull/1251/commits/755531f6d8693bcd0e4717457c3cbaa2160475ab